### DISCUSSION TITLE
fonts and themes

### DISCUSSION THREAD

**Discussion by FernandaMaria28** [Schweinepriester/github-profile-achievements#491] [Q&A] [answered]
What text fonts and themes do you recommend?

**Comment by jaenfigueroa** [comment, marked as answer]
I recommend you this web. it is called nerdfonts, here is the link https://www.nerdfonts.com/font-downloads , and here I leave you the repository in github https://github.com/ryanoasis/nerd-fonts , there are many good text fonts for vscode, it has ligatures, that will make your work much easier, many of them I have used, among my favorites are Jetbrains mono, fira code mono, hack nerd font, in themes for vscode, I recommend One Dark, Dracula, Community Material, SynthWave '84, here is a web: https://vscodethemes. com/ where you can find all the variety of themes available, try and find the one you like, install it and that's it.

**Reply by FernandaMaria28** [reply]
thank you very much, it helps me a lot

**Comment by bawejagb** [comment]
I recommend you to use the **JellyFish Theme** Extention, it has very great text fonts and interface. ❤️‍🔥
Must Try, I've been using it for the past few months, and I love it.
Link: [https://marketplace.visualstudio.com/items?itemName=PawelBorkar.jellyfish](url)
